src/main/java/org/broadinstitute/hellbender/tools/exome/HetPulldownCalculator.java

```diff
@@ -176,7 +185,7 @@ public Pulldown getTumor(final File tumorBAMFile, final IntervalList normalHetIn
      */
     private Pulldown getHetPulldown(final File bamFile, final IntervalList snpIntervals, final SampleType sampleType,
                                     final double pvalThreshold) {
-        try (final SamReader bamReader = SamReaderFactory.makeDefault().open(bamFile);
+        try (final SamReader bamReader = samReaderFactory.open(bamFile);
```

Comment:
The canonical form for this would be something like:

bamReader = SamReaderFactory.makeDefault().setValidationStringency(VALIDATION_STRINGENCY).setReference(this.refFile).open(bamFile)

This doesn't rely on any cached global state, and also passes in the reference file, which allows it to work on CRAM files as well.